how-to-guides/convert-pytorch-to-ignite.ipynb

```diff
@@ -0,0 +1,377 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "zv2kceT0CS-L"
+      },
+      "source": [
+        "def validation_step(engine, batch):\n",
+        "    model.eval()\n",
+        "    with torch.no_grad():\n",
+        "        x, y = batch\n",
+        "        y_pred = model(x)\n",
+        "\n",
+        "    return y_pred, y\n",
+        "    \n",
+        "    \n",
+        "evaluator = Engine(validation_step)"
+      ],
+      "execution_count": 31,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "EAIBqfFm8oqS"
+      },
+      "source": [
+        "Similar to the training loop, there are [helper methods](https://pytorch.org/ignite/engine.html#helper-methods-to-define-supervised-trainer-and-evaluator) to avoid writing this custom evaluation function like [`create_supervised_evaluator`](https://pytorch.org/ignite/generated/ignite.engine.create_supervised_evaluator.html#ignite.engine.create_supervised_evaluator).\n",
+        "\n",
+        "**Note**: You can create different evaluators for training, validation, and testing if they serve different purposes. A common practice is to have two separate evaluators for training and validation, since the results of the validation evaluator are helpful in determining the best model to save after training."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "4t4PsYXn8ost"
+      },
+      "source": [
+        "## Switch to built-in Metrics\n",
+        "\n",
+        "Then we can replace the code for calculating metrics like accuracy and instead use several [out-of-the-box metrics](https://pytorch.org/ignite/metrics.html#complete-list-of-metrics) that Ignite provides or write a custom one (refer [here](https://pytorch.org/ignite/metrics.html#how-to-create-a-custom-metric)). The metrics will be computed using the `evaluator`'s output. Finally, we attach these metrics to the `evaluator` by providing a key name (\"accuracy\" below) so they can be accessed via `engine.state.metrics[key_name]`."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "iUVAOP6kFdA-"
+      },
+      "source": [
+        "from ignite.metrics import Accuracy\n",
+        "\n",
+        "Accuracy().attach(evaluator, \"accuracy\")"
+      ],
+      "execution_count": 8,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "WnGK925N5AR7"
+      },
+      "source": [
+        "## Organising code into Events and Handlers\n",
```

Comment:
Organising vs Organizing, I imagine it is British vs American. Let's use "z" as otherwise looks like a typo. What do you think ?